token/src/native_token.rs

```diff
@@ -0,0 +1,3 @@
+
+/// The token type for wrapped native SOL
+solana_sdk::declare_id!("So11111111111111111111111111111111111111111");
```

Comment:
There's no `Mint` account for wrapped SOL.   But the `Account::token` value is set to `So11111111111111111111111111111111111111111` so that the `Transfer` instruction can do the right thing if it's wrapped and for RPC inspection.